src/timetrackings/types.ts

```diff
@@ -1,4 +1,7 @@
 export interface TimeTracking {
```

Comment:
Later on this one will have to get readdressed, because I believe that our API either returns null or a value.

As following types seem that are not returned from API:
  ```
start_time__timezone_hours_to_add?: string;
end_time__timezone_hours_to_add?: string;
is_offline_tracking?: boolean;
insertIntoConflicting?: string;
entityToCreateOnSplit?: string;
_writePermissionType?: string;
```

Reply:
ok, I will make a ticket to address this.

Reply:
SB-819